Source/Turbo/ViewControllers/HotwireTabBarController.swift

```diff
@@ -4,7 +4,17 @@ import UIKit
 ///
 /// This controller loads tabs defined by `HotwireTab` and configures each one with its own `Navigator`.
 /// The currently selected tab's navigator is exposed via the `activeNavigator` property.
-open class HotwireTabBarController: UITabBarController {
+open class HotwireTabBarController: UITabBarController, Router {
+    public init(navigatorDelegate: NavigatorDelegate? = nil) {
+        self.navigatorDelegate = navigatorDelegate
+        super.init(nibName: nil, bundle: nil)
+    }
```

Comment:
This separates the loading of the tabs with the assigning of the delegate. I'm using this approach to assign a delegate once then dynamically update/load tabs later, from somewhere else in the codebase.